sdk/include/reactos/ui/rosmsstyles.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * PROJECT:     ReactOS headers
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
+ * PURPOSE:     Support for PNG images in visual styles
+ * COPYRIGHT:   Copyright 2023 Ethan Rodensky <Splitwirez@gmail.com>
+ */
+
+#if (DLL_EXPORT_VERSION >= _WIN32_WINNT_LONGHORN) && !defined(MSSTYLES_ACCEPT_PNG_IMAGES)
+// Dictates whether or not uxtheme will use whatever PNG images are present in a visual style
+#define MSSTYLES_ACCEPT_PNG_IMAGES
+#endif
+
+#if defined(MSSTYLES_ACCEPT_PNG_IMAGES) && !defined(MSSTYLES_INCLUDE_PNG_IMAGES)
+// Dictates whether or not built-in visual styles (media/themes/**/*.msstyles) are permitted to include PNG images
+#define MSSTYLES_INCLUDE_PNG_IMAGES
```

Comment:
I don't see why this should be two defines, why not reduce it to just one? Also why **ros**msstyles.h ?

Reply:
If other headers in this directory are named similarly, then it's fine. Reducing the ifdefs would still be preferable, unless someone provides constructive points for having more than one.